README.md

```diff
@@ -2,96 +2,33 @@
 
 # Polkadot Secure Validator Setup
 
-This repo describes a potential setup for a Polkadot validator that aims to prevent
-some types of potential attacks.
+This repo describes a potential setup for a Polkadot validator that aims to
+prevent some types of potential attacks. The [Workflow](#workflow) section
+describes the [Platform Layer](#platform-layer) and the [Application
+Layer](#application-layer) in more detail.
 
-## How to use
+![Polkadot Secure Network Chart](secure_network_chart.svg)
 
-This repo has code for creating a complete implementation of the approach
-described [here](https://hackmd.io/QSJlqjZpQBihEU_ojmtR8g) from scratch, including
-both layers described in [Workflow](#workflow). This can be done on a host with
-NodeJS, Yarn and Git installed with:
+## Usage
 
-### Prerequisites
+There are two ways of using this repository:
 
-Before using polkadot-secure-validator you need to have installed:
+* **Application Layer**
 
-* NodeJS (we recommend using [nvm](https://github.com/nvm-sh/nvm))
+  Setup Debian-based machines yourself, which only need basic SSH access and
+  configure those in an inventory. The Ansible scripts will setup the entire
+  [Application Layer](#application-layer).
 
-* [Yarn](https://yarnpkg.com/lang/en/docs/install)
+  See the [Ansible Guide](GUIDE_ANSIBLE.md) for more.
 
-* [Terraform](https://www.terraform.io/downloads.html) (the snap package available via your package manager will not work)
+* **Platform & Application Layer** [ADVANCED]
```

Comment:
I think we should move this section before ** Application Layer ** as the first way of using this repository, and remove the `[ADVANCED]`, it depends on the users' capabilities (won't be advanced for everyone).